explanation/intro-to/cloud-init.md

```diff
@@ -0,0 +1,50 @@
+(introduction-to-cloud-init)=
+
+# Introduction to cloud-init
+
+Managing and configuring cloud instances and servers can be a complex and
+time-consuming task. [Cloud-init](https://docs.cloud-init.io/en/latest/ ) is
+the industry-standard open source tool designed to automate getting systems up
+and running with preconfigured settings in a reliable and repeatable way across instances.
+
+Although it’s commonly used to automatically configure public or private cloud
+instances, it can also be used to deploy virtual machines and physical machines
+on a network. By automating all the routine setup tasks, systems can be
+initialized efficiently and reliably, whether you’re a developer spinning up
+virtual machines or containers, or a system administrator managing
+infrastructure.
+
+## How does it work?
+
+Cloud-init works by taking the initial configuration that you supply, and
+applying it at boot time so that when the instance is launched it’s already
+configured the way you want.
+
+Your configuration can be used and re-used as often as you want - so that
+whether you want the exact same VM environment every time, or want to deploy an
+entire fleet of machines in exactly the same way, you will get reliable and
+repeatable results with a fraction of the time and effort it would take to do
+so manually.
```

Comment:
```suggestion
Your configuration can be used and re-used as often as you want to get the exact same VM environment every time, or to deploy an
entire fleet of machines in exactly the same way. You get consistent results with a fraction of the time and effort it would take to do
so manually.
```